synapse/lib/cell.py

```diff
@@ -2410,22 +2425,31 @@ def modCellConf(self, conf):
         '''
         for key, valu in conf.items():
             self.conf.reqKeyValid(key, valu)
+            logger.info(f'Setting cell config override for [{key}]')
 
         self.conf.update(conf)
-        s_common.yamlmod(conf, self.dirn, 'cell.yaml')
+        s_common.yamlmod(conf, self.dirn, 'cell.mods.yaml')
 
     def popCellConf(self, name):
         '''
-        Remove a key from the Cell's ondisk configuration file.
+        Remove a key from the Cell's ondisk configuration overrides file and
+        runtime configuration.
 
         Args:
             name (str): Name of the value to remove.
 
+        Notes:
+            This does re-validate the configuration after removing the value,
+            so if the value removed had a default populated by schema, that
+            default would be reset.
```

Comment:
Might be good to add to this note the fact that you cannot popCellConf() for any value except one that was already set with modCellConf() ( since we didn't go with a sentinel value )